docs/part1.md

```diff
@@ -187,19 +185,15 @@ Finally, remove all data with additional elements by selecting the **"only compo
 
 Actually, the **order** in which we applied the filters **does not matter**.
 In that sense, filters are like small questions or details that we use to construct an overall query.
-This insight comes with a small caveat: the **statistics** that we see as we are stacking up our filters **are contingent on the history**.
-If we take a different route, we will see different intermediate results and this could lead us to decide on different filters.
-
-To see this effect in action, rewind back to where we were choosing our priorities and let us go with the latter instead.
-The fastest way here, is to retrieve **"Molecular Dynamics" > "Available Properties"** and **click the highlighted checkbox** again to deselect it.
-
+<!-- This insight comes with a small caveat: the **statistics** that we see as we are stacking up our filters **are contingent on the history**.
+If we take a different route, we will see different intermediate results and this could lead us to decide on different filters. -->
+Let's go back to when we were selecting the relevant option under the Molecular Dynamics filter (click the `x` next to the `temperature chip` or go to **"Molecular Dynamics" > "Available Properties"** and **click the highlighted checkbox** again to deselect it).
 
 ??? success
     Glossing over the remaining 6 results, 3 by Andrew Rosen and 3 by Sebastian Baugmart, we retrieve **2 more MD simulations** and **2 more "Geometry Optimizations"**.
-    Even though some entries do not have a clear specification, we can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
-    While geometry optimization was not part of match our initial objective, these entries could still provide us with valuable **insights into the forcefield** setup.
-    They also inform us of the **overarching workflow** used by both authors, .e., relaxing the system before the production simulation.
-    Annotating such workflows is covered more extensively in [part II](part2.md).
+    You may notice that some entries do not have a clear specification. These entries likely need to be reprocessed to classify them correctly, since the molecular dynamics support has been developed more recently. However, you can find their classification by visiting the entry's overview page. <!-- @Joseph is this a bug? -->
+    While geometry optimization was not part of match our initial objective, these entries may inform us about the **overarching workflow** used by both authors, .e., relaxing the system before the production simulation.
```

Comment:
i.e., relaxing the system before the production simulation.

Reply:
👍